packages/types/src/package.ts

```diff
@@ -131,6 +131,9 @@ export type ProjectManifest = BaseManifest & {
     allowedDeprecatedVersions?: AllowedDeprecatedVersions
     allowNonAppliedPatches?: boolean
     patchedDependencies?: Record<string, string>
+    update?: {
+      ignore?: string[]
```

Comment:
I defined `pnpm.update.ignore` as string array. In the origin issue, `pnpm.update.ignore` is defined as `Record<string, string>`, like `package: version`. However, I think filtering ignoreing package with version is not necessary and it has performance overhead to get current lockfile in the process of `pnpmCommands.handler`, so I defined `pnpm.update.ignore` as string array.

Reply:
I would call it `ignoreDependencies`

Reply:
`ignoreDependencies` seems more clear, thanks! [99b4197](https://github.com/pnpm/pnpm/pull/5408/commits/99b4197424f5b0e34e24b8a9fe458dc5e738a7f1)